src/lib/conf/confmacros.h

```diff
@@ -61,7 +61,8 @@
 
 #define CONFIG_VAR_OBSOLETE(varname)            \
   { .member = { .name = varname, .type = CONFIG_TYPE_OBSOLETE },        \
-    .flags = CVFLAG_OBSOLETE                                            \
+    .flags = CFLG_NOSET | CFLG_NOLIST | CFLG_NODUMP | CFLG_NOCOPY |     \
+             CFLG_NOCMP                                                 \
```

Comment:
This group of flags is duplicated in at least two places: let's define them in one place, using a macro?